apps/right-to-rent-check/index.js

```diff
@@ -179,6 +179,9 @@ module.exports = {
       fields: ['tenancy-start'],
       next: '/check-confirmed'
     },
+    '/check-not-needed-date': {
+      backLink: 'tenancy-start-date'
```

Comment:
This is configured to work without the `/` e.g. `/tenancy-start-date`.  I think that seems a bit strange considering next requires `/`.  I will put in a PR for this

Reply:
Good luck finding the code for that ;-)